src/crate/client/sqlalchemy/dialect.py

```diff
@@ -194,8 +195,16 @@ def connect(self, host=None, port=None, *args, **kwargs):
             server = '{0}:{1}'.format(host, port or '4200')
         if 'servers' in kwargs:
             server = kwargs.pop('servers')
-        if server:
-            return self.dbapi.connect(servers=server, **kwargs)
+        servers = to_list(server)
+        if servers:
+            # Evaluate "ssl" connection URI query parameter.
+            # TODO: Evaluate more parameters like `ssl_ca`, `ssl_key`, `ssl_cert`,
+            #       `ssl_capath` and `ssl_cipher`.
+            if "ssl" in kwargs:
+                use_ssl = asbool(kwargs.pop("ssl"))
+                if use_ssl:
```

Comment:
```suggestion
              use_ssl = asbool(kwargs.pop("ssl"))
              if use_ssl:
```

Looks like this should work too

Reply:
I don't think it will work but raise an exception instead, when `ssl` is not available within the parameter dictionary. Am I missing something?
```
>>> a = {}
>>> a.pop("b")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
KeyError: 'b'
```

Reply:
Right, would have to use `kwargs.pop("ssl", False)` or with `None`

Reply:
Ah, right. Addressed with f1b22382f5.